nucliadb_models/src/nucliadb_models/common.py

```diff
@@ -271,3 +277,7 @@ class Answer(BaseModel):
 class QuestionAnswer(BaseModel):
     question: Question
     answers: List[Answer]
+
+
+class QuestionAnswers(BaseModel):
```

Comment:
Do we need a new type only for a list of `QuestionAnswer`?

Reply:
As a default pattern map the PB